Resources/Textures/Constructible/Atmos/gasgenerator.rsi/meta.json

```
@@ -0,0 +1,15 @@
+{
+   "version":1,
+   "size":{
+      "x":32,
+      "y":32
+   },
+   "license":"CC-BY-SA-3.0",
+   "copyright":"Taken from https://github.com/tgstation/tgstation at commit 57cd1d59ca019dd0e7811ac451f295f818e573da",
+   "states":[
+      {
+         "name":"gasGenerator",
+         "directions":1
```

Nitpicky but directions not required anymore if it's 1.

```suggestion
         "name":"gasGenerator"
```